File: app/src/main/java/com/caiqueponjjar/cuco/Item.java
package com.caiqueponjjar.cuco;

public class Item {
    private String itemTitle;
    private String itemAuthor;

    public Item(String itemTitle, String itemAuthor) {
        this.itemTitle = itemTitle;
        this.itemAuthor = itemAuthor;
    }

    public String getItemTitle() {
        return itemTitle;
    }

    public String getItemSubtitle() {
        return itemAuthor;
    }
}

File: app/src/main/java/com/caiqueponjjar/cuco/Item.kt
package com.caiqueponjjar.cuco;

import android.graphics.Color

public class Item {
    var itemTitle : String = " "
    var itemSubtitle : String = " "

    var itemColor : Int = Color.parseColor("#E05F22");
    var itemKey : String = " "
    constructor(itemTitle: String, itemSubtitle : String, itemColor : Int, itemKey : String) {
        this.itemTitle = itemTitle
        this.itemSubtitle = itemSubtitle
        this.itemKey = itemKey
        this.itemColor = itemColor
    }
}